app-antivirus/clamav/clamav-1.2.0_rc.ebuild

```
@@ -0,0 +1,398 @@
+	hashbrown@0.12.3
+	heck@0.4.1
+	hermit-abi@0.1.19
+	hermit-abi@0.3.2
+	hex@0.4.3
+	image@0.24.6
+	indexmap@1.9.3
+	instant@0.1.12
+	io-lifetimes@1.0.11
+	itoa@1.0.8
+	jpeg-decoder@0.3.0
+	js-sys@0.3.64
+	lazy_static@1.4.0
+	lazycell@1.3.0
+	lebe@0.5.2
+	libc@0.2.147
+	libloading@0.7.4
+	linux-raw-sys@0.3.8
+	lock_api@0.4.10
+	log@0.4.19
+	memchr@2.5.0
+	memoffset@0.9.0
+	minimal-lexical@0.2.1
+	miniz_oxide@0.7.1
+	nanorand@0.7.0
+	nom@7.1.3
+	num-complex@0.4.3
+	num-integer@0.1.45
+	num-rational@0.4.1
+	num-traits@0.2.15
+	num_cpus@1.16.0
+	once_cell@1.18.0
+	os_str_bytes@6.5.1
+	peeking_take_while@0.1.2
+	pin-project@1.1.2
+	pin-project-internal@1.1.2
+	png@0.17.9
+	prettyplease@0.2.10
+	primal-check@0.3.3
+	proc-macro2@1.0.64
+	qoi@0.4.1
+	quote@1.0.29
+	rayon@1.7.0
+	rayon-core@1.11.0
+	redox_syscall@0.3.5
+	regex@1.9.1
+	regex-automata@0.3.3
+	regex-syntax@0.7.4
+	rustc-hash@1.1.0
+	rustdct@0.7.1
+	rustfft@6.1.0
+	rustix@0.37.23
+	ryu@1.0.14
+	scopeguard@1.1.0
+	serde@1.0.171
+	serde_derive@1.0.171
+	serde_json@1.0.102
+	sha1@0.10.5
+	sha2@0.10.7
+	shlex@1.1.0
+	simd-adler32@0.3.5
+	smallvec@1.11.0
+	spin@0.9.8
+	strength_reduce@0.2.4
+	strsim@0.10.0
+	syn@1.0.109
+	syn@2.0.25
+	tempfile@3.6.0
+	termcolor@1.2.0
+	textwrap@0.16.0
+	thiserror@1.0.43
+	thiserror-impl@1.0.43
+	tiff@0.8.1
+	toml@0.5.11
+	transpose@0.2.2
+	typenum@1.16.0
+	unicode-ident@1.0.10
+	unicode-segmentation@1.10.1
+	version_check@0.9.4
+	wasi@0.11.0+wasi-snapshot-preview1
+	wasm-bindgen@0.2.87
+	wasm-bindgen-backend@0.2.87
+	wasm-bindgen-macro@0.2.87
+	wasm-bindgen-macro-support@0.2.87
+	wasm-bindgen-shared@0.2.87
+	weezl@0.1.7
+	which@4.4.0
+	winapi@0.3.9
+	winapi-i686-pc-windows-gnu@0.4.0
+	winapi-util@0.1.5
+	winapi-x86_64-pc-windows-gnu@0.4.0
+	windows-sys@0.48.0
+	windows-targets@0.48.1
+	windows_aarch64_gnullvm@0.48.0
+	windows_aarch64_msvc@0.48.0
+	windows_i686_gnu@0.48.0
+	windows_i686_msvc@0.48.0
+	windows_x86_64_gnu@0.48.0
+	windows_x86_64_gnullvm@0.48.0
+	windows_x86_64_msvc@0.48.0
+	zune-inflate@0.2.54
+"
+
+inherit cargo cmake flag-o-matic llvm python-any-r1 systemd tmpfiles
+
+MY_P=${P//_/-}
+
+DESCRIPTION="Clam Anti-Virus Scanner"
+HOMEPAGE="https://www.clamav.net/"
+SRC_URI="https://github.com/Cisco-Talos/clamav/archive/refs/tags/${MY_P}.tar.gz
+	${CARGO_CRATE_URIS}"
+S=${WORKDIR}/clamav-${MY_P}
+
+LICENSE="Apache-2.0 BSD GPL-2 ISC MIT MPL-2.0 Unicode-DFS-2016 ZLIB"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos"
+
+IUSE="doc clamonacc +clamapp experimental jit libclamav-only milter rar selinux systemd test"
+
+REQUIRED_USE="libclamav-only? ( !clamonacc !clamapp !milter )
+	clamonacc? ( clamapp )
+	milter? ( clamapp )
+	test? ( !libclamav-only )"
+
+RESTRICT="!test? ( test )"
+
+# Require acct-{user,group}/clamav at build time so that we can set
+# the permissions on /var/lib/clamav in src_install rather than in
+# pkg_postinst; calling "chown" on the live filesystem scares me.
+CDEPEND="
```

```suggestion
COMMON_DEPEND="
```
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Avoid_using_single-char_.2ADEPEND_variable